src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -2160,6 +2160,9 @@ public static final void updateCombatData(
         }
         case AdventurePool.WARBEAR_FORTRESS_LEVEL_THREE -> ResultProcessor.processItem(
             ItemPool.WARBEAR_BADGE, -1);
+        case AdventurePool.SHADOW_RIFT -> {
+          Preferences.increment("_shadowRiftCombats");
```

Comment:
```
      int adventure = KoLAdventure.lastAdventureId();
...
      switch (adventure) {
```
If you are adventuring in a Shadow Rift, your lastVisitedLocation will be, for example "Shadow Rift (Hidden Temple)" and the "adventureId" will be "place=shadow_rift". Since that is non-numeric, KoLAdventure.lastAdventureId() returns 0. Therefore, this code is unreachable with Shadow Rift adventure handling as written.